src/amp/server/render.tsx

```diff
@@ -48,13 +50,18 @@ export const render = ({ body }: express.Request, res: express.Response) => {
             canonicalURL: CAPI.webURL,
         };
 
+        const experimentsData: AmpExperiments =
+            await fetch(ampExperimentsDataUrl)
```

Comment:
This will make a request for each page render. I don't think we want to introduce a network request here as it slows down rendering (and we have to go back to the event loop).

Instead, let's independently poll the endpoint and cache the data. This can be done with `setTimeout`, and it's ok if `experimentsData` is empty when the server first starts up

Reply:
Good point, thanks @tomrf1!